modal/functions.py

```diff
@@ -1178,7 +1178,7 @@ async def _map(
         if self._web_url:
             raise InvalidError(
                 "A web endpoint function cannot be directly invoked for parallel remote execution. "
-                f"Invoke this function via its web url '{self._web_url}' or call it locally: {self._function_name}()."
```

Comment:
I think it's a different issue — just outdated prose from the days when you could invoke a local function by calling it directly. I *think* (but haven't checked) that `.local()` should still work, at least for web endpoints. But probably not, or at least not the way you expect, for `@asgi_app` and friends. So I think I'm fine with omitting any mention of local execution.

Reply:
I changed it to `.local()` so it at least is not wrong. Technically it works for `@asgi_app` as well, but it will return the actual app, not sure what the use case would be but at least it's possible 🤷